### DISCUSSION TITLE
What problems will I encounter if I use sqLite to implement the offline search function?

### DISCUSSION THREAD

**Discussion by HelloTrigger** [organicmaps/organicmaps#8664] [Q&A]
I have been researching offline map search solutions recently. Organic map has developed a set of indexing and querying methods for data. I would like to know if there would be any issues if I implement this indexing process using sqLite.

Tnanks.

**Comment by biodranik** [comment]
Your search may be slow. Or data will use more space/memory.

**Reply by HelloTrigger** [reply]
> Your search may be slow. Or data will use more space/memory.

Thank you for your reply. 
I am currently conducting research on offline search engines.
Do you have any comparative data? Or do you have an estimate of how much performance will decrease? What if I split some tables based on Rect?
